lib/src/androidTest/java/com/orange/ods/compose/component/appbar/top/OdsTopAppBarTest.kt

```diff
@@ -0,0 +1,199 @@
+/*
+ * Software Name: Orange Design System
+ * SPDX-FileCopyrightText: Copyright (c) Orange SA
+ * SPDX-License-Identifier: MIT
+ *
+ * This software is distributed under the MIT license,
+ * the text of which is available at https://opensource.org/license/MIT/
+ * or see the "LICENSE" file for more details.
+ *
+ * Software description: Android library of reusable graphical components
+ */
+
+package com.orange.ods.compose.component.appbar.top
+
+import androidx.activity.ComponentActivity
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.automirrored.filled.ArrowBack
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.test.assertIsDisplayed
+import androidx.compose.ui.test.assertIsEnabled
+import androidx.compose.ui.test.assertIsNotDisplayed
+import androidx.compose.ui.test.assertIsNotEnabled
+import androidx.compose.ui.test.junit4.createAndroidComposeRule
+import androidx.compose.ui.test.onNodeWithText
+import androidx.compose.ui.test.performClick
+import com.orange.ods.compose.component.menu.OdsDropdownMenu
+import com.orange.ods.compose.extension.setOdsContent
+import com.orange.ods.compose.test.isDropdownMenuItem
+import com.orange.ods.compose.test.isDropdownMenuItemDivider
+import com.orange.ods.compose.test.isTopAppBarActionButton
+import com.orange.ods.compose.test.isTopAppBarNavigationIcon
+import com.orange.ods.compose.test.isTopAppBarOverflowMenu
+import org.junit.Rule
+import org.junit.Test
+import org.mockito.Mockito.mock
+import org.mockito.Mockito.verify
+import org.mockito.kotlin.never
+
+internal class OdsTopAppBarTest {
+
+    @get:Rule
+    val composeTestRule = createAndroidComposeRule<ComponentActivity>()
+
+    @Test
+    fun odsTopAppBarRendersAsExpected() {
+        with(composeTestRule) {
+            val title = "Title"
+            val navigationIcon = OdsTopAppBar.NavigationIcon(Icons.AutoMirrored.Filled.ArrowBack, "Navigation icon") {}
+            lateinit var actions: List<OdsTopAppBar.ActionButton>
+            val overflowMenuItems = listOf(
+                OdsDropdownMenu.Item("Settings") {},
+                OdsDropdownMenu.Item("Account") {}
+            )
+
+            setOdsContent {
+                actions = listOf(
+                    OdsTopAppBar.ActionButton(painterResource(id = android.R.drawable.ic_dialog_info), "Info") {},
+                    OdsTopAppBar.ActionButton(painterResource(id = android.R.drawable.ic_dialog_email), "Email") {}
+                )
+                OdsTopAppBar(
+                    title = title,
+                    navigationIcon = navigationIcon,
+                    actions = actions,
+                    overflowMenuItems = overflowMenuItems
+                )
+            }
+
+            onNodeWithText(title).assertIsDisplayed()
+            onNode(isTopAppBarNavigationIcon(navigationIcon)).assertIsDisplayed()
```

Comment:
We can add an `assertContentDescriptionEquals` on this node too.

Reply:
Good idea 👍